### DISCUSSION TITLE
How do I make useContractRead async?

### DISCUSSION THREAD

**Discussion by raffle-x-dev** [wevm/wagmi#3559] [Q&A]
I need a parameter retrieved by a useContractRead hook.
Once I get that paramenter, I need to use it inside another hook.
I tried nesting the second hook inside the onSuccess(){}of the first hook but it doesn't work. :/

**Comment by lukasluecke** [comment]
https://tanstack.com/query/latest/docs/framework/react/guides/dependent-queries
